### ISSUE TITLE
Clean up parm/parm_wave directory and script input files

### ISSUE THREAD

**Issue by JessicaMeixner-NOAA** [NOAA-EMC/global-workflow#1244]
**Description**
This is a low-priority clean-up issue.  The files in parm/parm_wave  only differ based on one or two things: either forecast hour or grid features.  There's no need for so many files, however care must be taken to have the correct format in the end.  Note files could change in the future. 

**Requirements**
Less files 

**Acceptance Criteria (Definition of Done)**
We have less files and script the parm/parm_wave/* files to have less files and less oppertunity for mistakes if changes are made. 

**Dependencies**
None.

**Comment by arunchawla-NOAA** [comment]
@JessicaMeixner-NOAA what help do you need?

**Comment by JessicaMeixner-NOAA** [comment]
This is very low priority, but if we wanted to clean up the parm/parm_wave directory, we could have either 1 or number of grid template files to be used instead of  (# of output grids)x(# of forecast hours) files  - which are used here: https://github.com/NOAA-EMC/global-workflow/blob/develop/scripts/exgfs_wave_prdgen_gridded.sh      

It's okay if it doesn't happen, but I made that issue (I believe at @aerorahul's request) so we could keep track of this.

**Comment by arunchawla-NOAA** [comment]
we should clearly get rid of the old grids. As for grib2 header files, these were created for the different forecast hours from Boi. Perhaps we can have a smarter way of doing this

**Comment by JessicaMeixner-NOAA** [comment]
I don't think we currently have any old grids in there right now, but we likely will need changes for GFSv17 just not sure exactly what that will be at this moment.

**Comment by DavidHuber-NOAA** [comment]
Noting that I am encountering an issue with the parm files when running the `gfs_waveawipsgridded` job.  This job uses the `parm/wave/grib2_gfswave.<grid>.fHHH` files.  However, the files are only available through 120 forecast hours and at a 3 hour interval while WW3 outputs are available every hour through 120 hours and every 3 hours through 384 forecast hours.

Rather than opening a new issue to track this, I am going to expand this one.

**Comment by JessicaMeixner-NOAA** [comment]
> Noting that I am encountering an issue with the parm files when running the `gfs_waveawipsgridded` job. This job uses the `parm/wave/grib2_gfswave.<grid>.fHHH` files. However, the files are only available through 120 forecast hours and at a 3 hour interval while WW3 outputs are available every hour through 120 hours and every 3 hours through 384 forecast hours.
> 
> Rather than opening a new issue to track this, I am going to expand this one.

@DavidHuber-NOAA  I suspect this is the expected AWIPS output frequency.  

Tagging @AminIlia-NOAA for his awareness

**Comment by DavidHuber-NOAA** [comment]
@JessicaMeixner-NOAA Noted, thank you.  In that case, I will add some variables to `parm/config/gfs/config.waveawipsgridded` to set the AWIPS output frequency and range.  Removing bug label.

**Comment by JessicaMeixner-NOAA** [comment]
We should confirm with what's being done in v16 - but I suspect that's what's going on.

**Comment by DavidHuber-NOAA** [comment]
~OK, looking in `/lfs/h1/ops/prod/com/gfs/v16.3/gfs.20250109/00/wave/gridded`, I see gridded products all the way through hour 384, at a 1-hour cadence through FH 120, then a 3-hour cadence thereafter.~

**Comment by DavidHuber-NOAA** [comment]
Oh, wrong location... That's not AWIPS.

**Comment by DavidHuber-NOAA** [comment]
Looking in `/lfs/h1/ops/prod/com/gfs/v16.3/gfs.20250107/00/wave/wmo`, the gridded products are available at a 3-hour cadence through 72 hours, then a 6-hour cadence through forecast hour 180.  This agrees with what is currently in develop:
https://github.com/NOAA-EMC/global-workflow/blob/fea98f9b572e1ab2ceffab7a06d0bcc0298d59cf/scripts/exgfs_wave_prdgen_gridded.sh#L34-L35

**Comment by JessicaMeixner-NOAA** [comment]
Thanks for checking and confirming this @DavidHuber-NOAA
